package.json

```diff
@@ -83,7 +83,7 @@
         "viem": "2.17.1",
         "wagmi": "2.12.25",
         "web3": "^1.10.4",
-        "zod": "^3.23.8"
```

Comment:
Q: what is this new dependency?

Reply:
It's an object validation library, was the one breaking vercel because of some obscure compatibility issues just used another one to fix